arangod/RocksDBEngine/RocksDBReplicationContext.cpp

```diff
@@ -129,7 +129,8 @@ int RocksDBReplicationContext::bindCollection(
       ((_collection->name() != collectionIdentifier) &&
        std::to_string(_collection->cid()) != collectionIdentifier &&
        _collection->globallyUniqueId() != collectionIdentifier)) {
-    _collection = _trx->vocbase()->lookupCollection(collectionIdentifier);
+    _collection = _trx->vocbase()->lookupCollection(collectionIdentifier).get();
```

Comment:
Why are you not converting this stuff to std::shared_ptr as well ?

Reply:
I tried to keep the changes to a minimum otherwise there are too many files in the changeset
although I do agree that eventually raw pointers should be shared pointers